src/soundbase.cpp

```diff
@@ -166,12 +166,9 @@ QString CSoundBase::SetDev ( const QString strDevName )
         if ( !vsErrorList.isEmpty() )
         {
             // create error message with all details
-            QString sErrorMessage = "<b>" + tr ( "No usable " ) + strSystemDriverTechniqueName +
-                                    tr ( " audio device "
-                                         "(driver) found." ) +
+            QString sErrorMessage = "<b>" + QString ( tr ( "No usable %1 audio device found." ) ) .arg ( strSystemDriverTechniqueName ) +
```

Comment:
```suggestion
            QString sErrorMessage = "<b>" + QString ( tr ( "No usable %1 sound hardware found." ) ) .arg ( strSystemDriverTechniqueName ) +
```

"No usable CoreAudio sound hardware"/"No useable ASIO sound hardware" sounds strange too. Do we even need the API here?

Reply:
This error box is going to be copy/pasted by the user when they try to get help, hence all the error messages being listed.

Having the `strSystemDriverTechniqueName` will be useful to whoever reads the request.

(Does this ever appear?)

Reply:
Yes, It does appear for users without proper sound hardware and only ASIO4ALL.

Reply:
If it only appears with ASIO4ALL, then it shouldn't be in soundbase.cpp, it should be in windows/sound.cpp.  Again, not for this change....

Reply:
Probably. I think there was an issue for this feature and probably Volker added this as a hotfix.